python/ray/serve/_private/constants.py

```diff
@@ -118,6 +118,13 @@
 #: being marked unhealthy.
 PROXY_HEALTH_CHECK_UNHEALTHY_THRESHOLD = 3
 
+# The proxy must be in the draining status for
+# at least this amount of time before it can be marked as drained.
+# This gives time for the ALB to stop sending traffic to this proxy.
+PROXY_DRAINING_MIN_PERIOD_S = (
```

Comment:
nit: 
```suggestion
PROXY_MIN_DRAINING_PERIOD_S = (
```
verbally you would said "the minimum drain period"